files/scripts/syncd.sh

```diff
@@ -64,8 +64,13 @@ start() {
 
     debug "Warm boot flag: ${SERVICE} ${WARM_BOOT}."
 
-    # Don't flush DB during warm boot
-    if [[ x"$WARM_BOOT" != x"true" ]]; then
+    if [[ x"$WARM_BOOT" == x"true" ]]; then
+        # Leave a mark for syncd scripts running inside docker.
+        touch /host/warmboot/warm-starting
+    else
+        rm -f /host/warmboot/warm-starting
+
+        # Don't flush DB during warm boot
```

Comment:
Looks more straight if we change the comment to: "# Flush ASIC DB during cold restart/reboot"?

Reply:
Copy & Paste sins. Sharp eyes!